lib/deimos/schema_class/record.rb

```diff
@@ -7,6 +7,16 @@ module Deimos
   module SchemaClass
     # Base Class of Record Classes generated from Avro.
     class Record < Base
+      # :nodoc:
+      def ==(other)
+        comparison = other
+        if other.class == self.class
+          comparison = other.state
+        end
+
+        comparison.stringify_keys.except('payload_key') == self.state.except('payload_key')
```

Comment:
Would it make more sense to just override `state` so that `payload_key` isn't output using it?

Second question... why do we have a method `state` when we could just use `as_json`? Missed this in the earlier PR.

Reply:
huh, good question :| as_json would work perfectly

Reply:
Doing both - with `as_json`, it's not used anywhere else